ui/apps/platform/src/services/ReportsService.ts

```diff
@@ -124,3 +124,11 @@ export function createReportConfiguration(
             return response.data;
         });
 }
+
+export function deleteReportConfiguration(reportId: string): Promise<Record<string, never>> {
```

Comment:
Nit: in `services/types.ts` we have an `Empty` type that is an alias for this.

Reply:
Oh nice, I'll use that then